nim/tests/test_routes.py

```diff
@@ -0,0 +1,103 @@
+from nim.app import app
+import pytest
+import json
+
+
+@pytest.fixture
+def test_app():
+    return app.test_client()
+
+
+def test_index(test_app):
+    response = test_app.get('/', content_type='html/text')
+    assert response.status_code == 200
+
+
+def test_new_success(test_app):
+    new_game_response = test_app.post(
+        '/new',
+        data='{"min": 10, "max": 10, "piles": 4}',
+        content_type='application/json'
+    )
```

Comment:
You should also check the `status_code`.

Reply:
Addressed 🙂